adoc/admin-ses-integration.adoc

```diff
@@ -87,36 +91,45 @@ spec:
       readOnly: false
 *EOF*
 ----
-. Verify that the pod exists and its status: 
+//TODO: Where do I get the input for POD_NAME, CONTAINER_NAME and IMAGE_NAME?
+//Can I just input any names I like?
+//Where do i get information for RBD_POOL and the monitors and their ports?
+. Verify that the pod exists and check its status:
 +
 
 ----
 kubectl get pod
 ----
-. Once the pod is running, check the mounted volume: 
+. Once the pod is running, check the mounted volume:
 +
 
 ----
 kubectl exec -it CONTAINER_NAME -- df -k ...
 /dev/rbd1               999320      1284    929224   0% /mnt/rbdvol
 ...
 ----
+//TODO: Could you indicate what this output means?
```

Comment:
The df command output means RBD has successfully mounted in pod. The block device mounted on /mnt/rbdvol, where the directory is provided in the manifest under `mountPath`. 
---
Filesystem             1K-block    Used    Available Used%   Mounted on
/dev/rbd1               999320      1284    929224   0%          /mnt/rbdvol